CaptchaBot/StartUpExtensions.cs

```diff
@@ -16,20 +17,26 @@ public static IApplicationBuilder UseTelegramBotWebhook(this IApplicationBuilder
             var lifetime = services.GetRequiredService<IHostApplicationLifetime>();
 
             lifetime.ApplicationStarted.Register(
-                async () =>
+                () =>
                 {
                     var logger = services.GetRequiredService<ILogger<Startup>>();
                     var address = services.GetRequiredService<AppSettings>().WebHookAddress;
 
-                    logger.LogInformation("Removing webhook");
-                    await services.GetRequiredService<ITelegramBotClient>().DeleteWebhookAsync();
+                    async Task RemoveWebHook()
```

Comment:
Let's name it something like `ResetWebHook`

Reply:
A good catch. Done!